app/scripts/TrackRenderer.jsx

```diff
@@ -1119,21 +1464,37 @@ class TrackRenderer extends React.Component {
     return last;
   }
 
+  /** @param {number} movement */
   valueScaleMove(movement) {
+    if (!this.zoomStartPos) {
+      return;
+    }
     // mouse wheel from zoom event
     // const cp = pointer(event.sourceEvent, this.props.canvasElement);
     for (const track of this.getTracksAtPosition(...this.zoomStartPos)) {
       track.movedY(movement);
     }
 
-    this.zoomTransform = this.zoomStartTransform;
+    if (this.zoomStartTransform) this.zoomTransform = this.zoomStartTransform;
```

Comment:
Same, could be undefined. Maybe we just set to `this.zoomTransform` in the constructor?